src/plugins/intel_cpu/src/nodes/multinomial.cpp

```diff
@@ -0,0 +1,200 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "multinomial.hpp"
+
+#include <openvino/op/multinomial.hpp>
+
+#include "ie_ngraph_utils.hpp"
+#include "ie_parallel.hpp"
+#include "shape_inference/custom/multinomial.hpp"
+
+namespace ov {
+namespace intel_cpu {
+namespace node {
+
+Multinomial::Multinomial(const std::shared_ptr<ov::Node>& op, const GraphContext::CPtr& context)
+    : Node(op, context, MultinomialShapeInferFactory(op)) {
+    std::string errorMessage;
+    if (!isSupportedOperation(op, errorMessage)) {
+        IE_THROW(NotImplemented) << (errorMessage);
+    }
+
+    auto multinomial_op = as_type_ptr<op::v13::Multinomial>(op);
+    m_convert_type = multinomial_op->get_convert_type();
+    m_with_replacement = multinomial_op->get_with_replacement();
+    m_log_probs = multinomial_op->get_log_probs();
+    m_global_seed = multinomial_op->get_global_seed();
+    m_op_seed = multinomial_op->get_op_seed();
+
+    m_errorPrefix = "Multinomial node with name '" + getName() + "' ";
+}
+
+bool Multinomial::isSupportedOperation(const std::shared_ptr<const ov::Node>& op, std::string& errorMessage) noexcept {
+    try {
+        if (op->get_type_info() != op::v13::Multinomial::get_type_info_static()) {
+            errorMessage = "Only Multinomial operation from the opset13 is supported by the CPU plugin.";
+            return false;
+        }
+    } catch (...) {
+        return false;
+    }
+    return true;
+}
+
+void Multinomial::getSupportedDescriptors() {
+    if (getParentEdges().size() != 2) {
+        IE_THROW() << m_errorPrefix << "has incorrect number of input edges.";
+    }
+    if (getChildEdges().size() != 1) {
+        IE_THROW() << m_errorPrefix << "has incorrect number of output edges.";
+    }
+}
+
+void Multinomial::initSupportedPrimitiveDescriptors() {
+    auto probs_prc = getOriginalInputPrecisionAtPort(PROBS_PORT);
+    if (!one_of(probs_prc,
+                InferenceEngine::Precision::FP64,
```

Comment:
There is no plans to support this precision. Please remove.

Reply:
Removed!